### PR TITLE
update compatibility to 3.1

### PR DESCRIPTION
**Pull request by LoganWright**

### CODE REVIEW

File: Utilities/compatibility.sh [hunk 1]
```diff
@@ -42,20 +42,19 @@ fi
```
Comment (tanner0101) [inline Utilities/compatibility.sh]: name this one `compatibility-1.sh` ?

Comment (LoganWright) [inline Utilities/compatibility.sh]: that will break our urls that point to `compatibility.sh` was my thinking here, any thoughts on how that would be affected?

Comment (tanner0101) [inline Utilities/compatibility.sh]: when we merge i just need to make sure to update qt2.

Comment (LoganWright) [inline Utilities/compatibility.sh]: cool, it's ready to go, let's sync up and do get em both at once

**Pull request closed without merging**